yt/geometry/coordinates/cartesian_coordinates.py

```diff
@@ -314,7 +337,7 @@ def _ortho_pixelize(
         if np.any(finfo.nodal_flag):
             nodal_data = get_nodal_data(data_source, field)
             coord = data_source.coord.d
-            pixelize_cartesian_nodal(
+            mask = pixelize_cartesian_nodal(
```

Comment:
yeah, the diversity in call signatures is one of the reasons this took a little while to converge.
Cases where the buffer is an input are more general and allow multi-step or parallel pixelisation, and I think it would make more sense to have the mask as an input too.